lib/cryptoexchange/exchanges/eosex/market.rb

```diff
@@ -0,0 +1,12 @@
+module Cryptoexchange::Exchanges
+  module Eosex
+    class Market < Cryptoexchange::Models::Market
+      NAME='eosex'
+      API_URL = 'https://api.eosex.com/exchange/tickers'
+
+      def self.trade_page_url(args={})
+        'https://eosex.com/#/'
```

Comment:
fill in the args[:base] and args[:target] to point directly to the trading page
eg. https://eosex.com/#/exchange/ANTETRX